tet.rs/src/tetris/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod board;
+pub mod my_tetris;
```

Comment:
Tetris module should probably be merged with main.rs

Reply:
Refacto is a WIP !